easybuild/easyblocks/c/cuda.py

```diff
@@ -17,8 +17,13 @@
 @author: George Tsouloupas (Cyprus Institute)
 @author: Fotis Georgatos (Uni.lu)
 @author: Kenneth Hoste (Ghent University)
+@author: Damian Alvarez (Forschungszentrum Juelich)
 """
 import os
+import stat
+import shutil
+
+from easybuild.framework.easyconfig import CUSTOM
```

Comment:
same here, order alphabetically in between the other `easybuild` imports below please